core/src/apps/solana/transaction/programs.json

```diff
@@ -0,0 +1,3228 @@
+{
```

Comment:
We are just currently implementing translations into firmware, and this file has quite a lot of hardcoded English strings (which will be shown to user).

It is an open question of how (and if - @Hannsek) we will support multiple languages for `Solana`

Do you have some ideas?

Reply:
I believe we should support it the same as other chains. So the less hardcoded English strings the better.

Reply:
Is there currently any action required on this from our side?

Reply:
we will need to replace the strings here with translation symbols and make them part of the master json.
(nothing to do for this PR though)